packages/terra-abstract-modal/tests/wdio/abstract-modal-spec.js

```diff
@@ -1,71 +1,70 @@
+/* global $ */
```

Comment:
Whoops, we'll want to add this to our eslint config (or whichever config) so this doesn't need to be specifically defined as a global in every spec

Reply:
https://github.com/cerner/terra-toolkit/blob/main/packages/eslint-config-terra/eslint.config.js#L74-L78